app/src/main/java/com/cornellappdev/android/eatery/WeeklyMenuFragment.java

```diff
@@ -0,0 +1,364 @@
+						if (lastClickedListView != null
+								&& lastExpandedPosition != -1
+								&& i != lastExpandedPosition) {
+							lastClickedListView.collapseGroup(lastExpandedPosition);
+						}
+						lastExpandedPosition = i;
+						lastClickedListView = expListViewWest;
+					}
+				});
+		expListViewNorth.setOnGroupExpandListener(
+				new NonScrollExpandableListView.OnGroupExpandListener() {
+					@Override
+					public void onGroupExpand(int i) {
+						if (lastClickedListView != null
+								&& lastExpandedPosition != -1
+								&& i != lastExpandedPosition) {
+							lastClickedListView.collapseGroup(lastExpandedPosition);
+						}
+						lastExpandedPosition = i;
+						lastClickedListView = expListViewNorth;
+					}
+				});
+
+		// Populate list of date TextViews on header
+		dateList.add((TextView) rootView.findViewById(R.id.date0));
+		dateList.add((TextView) rootView.findViewById(R.id.date1));
+		dateList.add((TextView) rootView.findViewById(R.id.date2));
+		dateList.add((TextView) rootView.findViewById(R.id.date3));
+		dateList.add((TextView) rootView.findViewById(R.id.date4));
+		dateList.add((TextView) rootView.findViewById(R.id.date5));
+		dateList.add((TextView) rootView.findViewById(R.id.date6));
+
+
+		// When changing date, highlight Breakfast textview
+		linDate.setOnClickListener(
+				new View.OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+						lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+					}
+				});
+
+		Date now = new Date();
+		Calendar cal = Calendar.getInstance();
+		cal.setTime(now);
+		for (int i = 0; i < 7; i++) {
+			// Formatting for each day
+			SimpleDateFormat dateFormat = new SimpleDateFormat("EEE");
+			dateFormat.setCalendar(cal);
+			String sDay = dateFormat.format(cal.getTime());
+
+			// Formatting for each date
+			int date = cal.get(Calendar.DAY_OF_MONTH);
+			String sDate = Integer.toString(date);
+			SpannableString ssDate = new SpannableString(sDay + '\n' + sDate);
+			ssDate.setSpan(new RelativeSizeSpan(0.8f), 0, 3, 0);
+			ssDate.setSpan(new RelativeSizeSpan(2f), 4, ssDate.length(), 0);
+			TextView tv = dateList.get(i);
+			tv.setText(ssDate);
+
+			cal.add(Calendar.DAY_OF_YEAR, 1);
+		}
+
+		breakfastText.setOnClickListener(
+				new View.OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+						lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
```

Comment:
might be worth declaring `getActivity().getApplicationContext()` as a variable to not write it out every time, and in the methods below